### DISCUSSION TITLE
uncouple backup and autoupdate please

### DISCUSSION THREAD

**Discussion by MAM-59** [nextcloud/all-in-one#5708] [Questions] [answered]
Hi, I don't need borg backup (the data is already on a "safe" place and backup otherwise), but I've noticed that without running the backup, the auto update of the dockers does not work too.
Currently I have to start the mastercontainer gui every few weeks and check for updates manually (OS tells me ALWAYS there would be updates but its lying).
This is not really nice.
Why are these two functions bundled and how can I split them up again?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-enable-automatic-updates-without-creating-a-backup-beforehand
